homeassistant/components/knx/cover.py

```diff
@@ -147,7 +148,12 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
 
     async def async_stop_cover(self, **kwargs: Any) -> None:
         """Stop the cover."""
-        await self._device.stop()
+        if self._device.supports_stop:
+            await self._device.stop()
+        elif self._device.is_opening():
+            await self._device.set_short_up()
+        elif self._device.is_closing():
+            await self._device.set_short_down()
```

Comment:
That's already handled in xknx
https://github.com/XKNX/xknx/blob/19bd73c8e68e7a537ccfba783f467703f55e98c8/xknx/devices/cover.py#L191